templates/header.hbs

```diff
@@ -1,19 +1,59 @@
-  <header class="header">
+<header class="header">
+  <nav class="header__container">
 
-    <nav class="header__container">
+    <a
+      class="header__logo"
+      href="/index.html"
+      aria-label="Logo Fundación Manao"
+    >
+      <img
+        src="/images/LogoManao.svg"
+        alt=""
+        role="presentation"
+        class="logo__image"
+      />
+    </a>
 
-      <a class="header__logo" href="/index.html" aria-label="Logo Fundación Manao">
-        <img src="/images/LogoManao.svg" alt="" role="presentation" class="logo__image">
-      </a>
-
-      <ul class="header__menu">
+    <ul class="header__menu">
       {{#each mainLinks}}
-        <li class="header__menu__item" >
+        <li class="header__menu__item">
           {{{module this}}}
         </li>
       {{/each}}
-      </ul>
+      <div class="header__menu__donor-info">
+        <h5>Donar para mas inclusión</h5>
```

Comment:
@jsebax Do we have this and all info below in the CMS?

Reply:
@JFozzie can you add classes to elements inside header__menu__donor-info?

Reply:
The donor component is not done yet, this info is temporal.